app/scripts/modules/core/src/application/config/DeleteApplicationSection.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+
+import { Application, ApplicationWriter } from 'core/application';
+import { ReactInjector } from 'core/reactShims';
+import { FirewallLabel } from 'core/securityGroup/label';
+
+export interface IDeleteApplicationSection {
+  application: Application;
+}
+
+export function DeleteApplicationSection(props: IDeleteApplicationSection) {
+  const { application } = props;
+
+  function deleteApplication(): void {
+    const taskMonitor = {
+      application,
+      title: 'Deleting ' + application.name,
+      hasKatoTask: false,
+      onTaskComplete: () => {
+        ReactInjector.$state.go('home.infrastructure');
+      },
+    };
+
+    ReactInjector.confirmationModalService.confirm({
+      header: 'Really delete ' + application.name + '?',
+      buttonText: 'Delete ' + application.name,
```

Comment:
nit: For all the string interpolation in this function you can use ES6 template literals.

```js
`Really delete ${application.name}?`
```

Reply:
👍